src/lib/get-topper-settings.js

```diff
@@ -10,8 +10,13 @@ const isNews = (content) =>
 	content.annotations &&
 	content.annotations.some((annotation) => annotation.prefLabel === 'News');
 
-const isLiveBlogOrPackage = (content) =>
-	(content.realtime && content.liveBlog) ||
+const isLiveBlogV1 = (content) =>
```

Comment:
I think a comment above this explaining that `live-blog` is a fake content type set by `next-article` for v1 live blogs would be useful, as otherwise there's no way of knowing where that's coming from i.e. it's not a `type` that's used in Elasticsearch.

Reply:
Ah yes! Good shout, I will do that

Reply:
Otherwise, we'll have this convo again in the future 😂